challenges-services/src/main/java/org/exoplatform/challenges/job/DataMigrationChallengeJob.java

```diff
@@ -135,10 +130,7 @@ public void execute(JobExecutionContext jobExecutionContext) throws JobExecution
                                 a.getComment(),
                                 a.getCreator(),
                                 a.getCreatedDate(),
-                                a.getActivityId(),
-                                creator.getRemoteId(),
-                                null,
-                                null);
+                                a.getActivityId());
```

Comment:
you should have added this Upgrade procedure as Data Upgrade Plugin and add it in Data-Upgrade plugin.
Please move this in dedicated addon and follow Best practices described here: https://community.exoplatform.com/portal/g/:spaces:engineering/exo_architects/notes/17080